### PR TITLE
Convert Delete Sitemap Entity Test

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
**Description**
This PR with a test that deletes Sitemap

**Steps:**
1. Log in as admin user from data set.
2. Navigate to Marketing > SEO and Search > Site Map.
3. Create Sitemap Entity
4. Find created sitemap in Grid
5. Delete Created Sitemap
6. Perform all Assertions

**Fixed issues:**
#339 Convert DeleteSitemapEntityTest to MFTF

### CODE REVIEW

File: app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <annotations>
+            <features value="Sitemap"/>
+            <stories value="Admin Deletes Sitemap Entity Test"/>
+            <title value="Sitemap Deleting Test"/>
+            <description value="Admin Should Delete Sitemap Entity"/>
+            <group value="sitemap"/>
+            <group value="mtf_migrated"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="LoginAsAdmin"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml]: `AdminLoginActionGroup`

File: app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            </actionGroup>
+            <!-- Navigate To New Sitemap Page -->
+            <actionGroup ref="AdminMarketingSiteMapNavigateNewActionGroup" stepKey="navigateToNewSitemapPage"/>
+            <!-- Create Sitemap Entity -->
+            <actionGroup ref="AdminMarketingSiteMapFillFormActionGroup" stepKey="createSitemap">
+                <argument name="sitemap" value="UniqueSitemapName"/>
+            </actionGroup>
+        </before>
+        <after>
+            <actionGroup ref="logout" stepKey="logout"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml]: `AdminLogoutActionGroup`

File: app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <actionGroup ref="AdminMarketingSiteDeleteByNameActionGroup" stepKey="deleteCreatedSitemap">
+                <argument name="filename" value="{{UniqueSitemapName.filename}}"/>
+            </actionGroup>
+            <actionGroup ref="AssertSiteMapDeleteSuccessActionGroup" stepKey="assertSuccessMessage"/>
+            <actionGroup ref="AdminMarketingSitemapSearchActionGroup" stepKey="searchDeletedSitemap">
+                <argument name="filename" value="{{UniqueSitemapName.filename}}"/>
+            </actionGroup>
+            <actionGroup ref="AssertAdminSitemapIsNotInGridActionGroup" stepKey="dontSeeEntity">
+                <argument name="filename" value="{{UniqueSitemapName.filename}}"/>
+            </actionGroup>
```
Comment (lbajsarowicz) [inline app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml]: Redundant indent.

File: app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml (hunk 1 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml]: Done

File: app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml (hunk 2 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml]: Done

File: app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml (hunk 3 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingDeleteSitemapEntityTest.xml]: Done

**Pull request merged**